metering/metering-upgrading-metering.adoc

```diff
@@ -0,0 +1,145 @@
+[id="upgrading-metering"]
+= Upgrading metering
+include::modules/common-attributes.adoc[]
+:context: upgrading-metering
+
+toc::[]
+
+You can upgrade metering to {product-version} by updating the Metering Operator subscription.
+
+[NOTE]
+====
+Upgrading the Metering Operator is supported for {product-title} 4.4.13 and later.
+====
+
+.Prerequisites
+
+*  The cluster is updated to {product-version}.
+*  The xref:../metering/metering-installing-metering.adoc#metering-install-operator_installing-metering[Metering Operator] is installed from OperatorHub.
++
+[NOTE]
+====
+You must upgrade the Metering Operator to {product-version} manually. Metering does not upgrade automatically if you selected the "Automatic" *Approval Strategy* in a previous installation.
+====
+*  The xref:../metering/configuring_metering/metering-about-configuring.adoc#metering-about-configuring[MeteringConfig] is configured.
+*  The xref:../metering/metering-installing-metering.adoc#metering-install-metering-stack_installing-metering[metering stack] is installed.
+*  Ensure that metering status is healthy by checking that all Pods are ready.
+
+[IMPORTANT]
+====
+Potential data loss can occur if you modify your metering storage configuration after installing or upgrading metering.
+====
+
+.Procedure
+
+.  Click *Operators* -> *Installed Operators* from the web console.
+
+.  Select the `openshift-metering` Project.
```

Comment:
Should "Project" be capitalized here? Not sure how the UI looks for this, but if it's referring to a project generically, I believe they're [typically lower-cased](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#project).

Reply:
Updated it to be lowercase. Good catch! I'll fix that in master/4.5/4.6 when I do the command/output split work for metering.